pithos/application.py

```diff
@@ -85,6 +87,38 @@ def do_command_line(self, command_line):
         # http://stackoverflow.com/questions/1943747/python-logging-before-you-run-logging-basicconfig
         logging.root.handlers = []
 
+        # Show the Pithos log since last reboot and exit
+        if options.contains('last-logs'):
+            try:
+                from systemd import journal
+                from os.path import basename
+            except ImportError:
+                type(command_line).do_print_literal(command_line, 'Systemd Python module not found')
```

Comment:
Missing a `\n` on all of these. Should move it to a helper function like so: https://github.com/lutris/lutris/blob/master/lutris/gui/application.py#L132-L135